test/io/ferguson/channel-comparision.chpl

```diff
@@ -0,0 +1,20 @@
+use IO;
```

Comment:
typo in filename -- should be:`channel-comparison.chpl` (update `.good` filename too)